Linear/main_LinEl.C

```diff
@@ -859,8 +859,8 @@ int main (int argc, char** argv)
 
     // Write load vector(s) to VTF-file
     const char* loadName[2] = { "Load vector", "Dual load vector" };
-    for (size_t i = 0; i < load.size() && i < 2; i++)
-      if (!model->writeGlvV(load[i],loadName[i],1,nBlock,2+i))
+    for (size_t l = 0; l < load.size() && l < 2; l++)
+      if (!model->writeGlvV(load[l],loadName[l],1,nBlock,2+l))
```

Comment:
I rather would use `j` instead of `i` then. I find `l` looking too much like the number `1` such that the chance of mis-reading is high if your sight is not too sharp.

Reply:
changed.